style.css

```diff
@@ -0,0 +1,97 @@
+body{
+    width: 90%;
+    height: 350px;
+    background-color: hsl(185, 75%, 39%);
+    background-image:url(bg-pattern-top.svg), url(bg-pattern-bottom.svg);
+    background-size: 450px, 450px;
+    background-repeat: no-repeat, no-repeat;
+    background-position: 50% 50%, 50% 50% ;    
+       
+    font-family: 'Kumbh Sans', sans-serif;
+    font-size: 18px;
+    
+    
+   
+}
+/*left -50px top -100px, right -50px bottom -400px;*/ 
+.container{
+    width: 250px;
+    height: 280px;
+    background-color: aliceblue;
+    border-radius: 8px;
+    margin-left: 475px;
+    margin-top: 180px;
+    
+}
+
+
+.a1 img{
+    width: 250px;
+    height: 100px;
+    border-top-left-radius: 8px;
+    border-top-right-radius: 8px;
+
+}
+
+.a2 img{
+    border-radius: 50px;
+    
+    border: 5px solid white;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: -50px;
+}
+
+
+.text{
+    font-size: 14px;
+    text-align: center;
+    line-height: 8px;
+}
+#victor{
+    line-height: 5px;
+}
+#london{
+   
+    font-size: 10px;
+}
+
+.container1{
+    position: relative;
+    
+
+}
+#p1{
+    font-family: 'Kumbh Sans', sans-serif;
+    font-size: 13px;
+    line-height: 5px;
+    position: absolute;
+    left: 35px;
+}
+#p2{
```

Comment:
Uopšteno se koriste klase umesto ID-ja. Na primer, napravio bi jednu klasu koja bi imala sve zajedničke stilove za #p1, #p2 i #p3, i onda bi dodao posebno stilove koji su posebni za svaki od elemenata:
```
.stats-wrapper {  
    font-family: 'Kumbh Sans', sans-serif;  
    font-size: 13px;  
    line-height: 5px;  
    position: absolute;  
}  
#p1 {
    left: 35px;  
}  
#p2 {
    left: 110px;
}
#p3 {
    right: 30px;
}
```
Dodao bi svakom od ovih elemenata klasu stats-wrapper u kojoj je zajednicki kod, a izbacio bi ga iz #p 
Neces to raditi ovde jer ces ovde koristiti display flex, kao sto smo pricali